src/error/__tests__/formatError-test.ts

```diff
@@ -8,8 +6,8 @@ import { GraphQLError } from '../GraphQLError';
 
 describe('formatError: default error formatter', () => {
   it('uses default message', () => {
-    // $DisableFlowOnNegativeTest
-    const e = new GraphQLError();
+    // @ts-expect-error
```

Comment:
I already enabled it on master but it requires comments please add some short like 
`negative test` or similar.

Reply:
yeah fixed, will push in a minute